projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-checkout.ts

```diff
@@ -171,9 +240,7 @@ export function selectAccountShippingAddress() {
 }
 
 export function selectAccountDeliveryMode() {
-  const getCheckoutDetails = interceptCheckoutB2BDetailsEndpoint(
```

Comment:
Address ID mismatch between mock Address obj and server address entity (PK value) makes test fails.
Using the server response rather than mock object to avoid ID this situation.